Wikipedia/Code/WMFAppViewController.m

```diff
@@ -761,6 +762,17 @@ - (BOOL)processUserActivity:(NSUserActivity *)activity completion:(dispatch_bloc
             [self.rootTabBarController setSelectedIndex:WMFAppTabTypeExplore];
             [[self navigationControllerForTab:WMFAppTabTypeExplore] popToRootViewControllerAnimated:NO];
             break;
+        case WMFUserActivityTypePlaces:
+        {
+            [self.rootTabBarController setSelectedIndex:WMFAppTabTypePlaces];
+            [[self navigationControllerForTab:WMFAppTabTypePlaces] popToRootViewControllerAnimated:NO];
+            NSURL *articleURL = activity.wmf_articleURL;
+            if (articleURL) {
+                [[self placesViewController] view]; // force view instantiation
+                [[self placesViewController] showArticleURL:articleURL];
```

Comment:
Is it possible to put the `[[self placesViewController] view]` (or something to achieve the same effect) in `showArticleURL`?